### ISSUE TITLE
Signs not appearing on the front of a chest

### ISSUE THREAD

**Issue by alchemistmatt** [AddstarMC/QuickShop#3]
In version 4.8.2, build 44, depending on the direction you are facing, the sign is placed differently on the chest.  To test this, place four different chests, but face a different direction (north, south, east, west) each time you place the chest.  Now,face the front of each test, sneak, and punch the chest with an item to sell. Enter the price in chat to sell the item.  Now, examine the sign.  Depending on which direction the chest was, the sign will be placed differently: sometimes on the chest, some times 90 degrees to the chest, sometimes 1 block away from the chest.

**Comment by Narimm** [comment]
Can I ask if the sign doesnt get placed on the chest does it always face the same cardinal direction:

**Comment by NeverGrowUp** [comment]
Yes it does. Because the sign is placed in the next blockspace of a chest, it is placed on the southern side of that blockspace.

**Comment by alchemistmatt** [comment]
I ran a new test with QuickShop 4.8.2, build 45, placing four chests, each facing a different direction.  The chest that faces north works; the sign is located on the front of the chest and it shows the seller, the stock, the item name, and the cost.  Furthermore, I am able to successfully purchase items from that quickshop.

On the other three chests the sign appears incorrectly (see the screenshot below).  Note that in all four cases the sign is facing south.
![quickshopsigns_build45](https://user-images.githubusercontent.com/6363449/49691397-1241c880-faf6-11e8-9414-8c2369a815ac.png)
(The fact that the chests are staying open is a client bug, new to 1.13; that's not QuickShop's fault.)

**Comment by alchemistmatt** [comment]
I'm not sure if this matters, but when I break a buggy QuickShop chest it says "Shop removed" in green text.  Then, when I break the sign that was next to the buggy chest, it says "The leaderboard was removed." in white text; that message is coming from some other plugin we have installed.  In contrast, removing the functioning QuickShop chest says only "Shop removed" in green text.

One more interesting note: you can buy items from the buggy quickshop chests: Left click the front of the chest, and it asks how many items you want to buy.  Enter a count in chat, and you successfully buy the items.  The sign near the buggy chest remains blank.
